src/vsr/replica.zig

```diff
@@ -4917,6 +4919,14 @@ pub fn ReplicaType(
                     self.send_prepare_oks_after_view_change();
                 }
             }
+
+            if (self.status == .view_change and self.log_view < self.view) {
+                if (self.primary_index(self.view) != self.replica and
+                    self.start_view_change_quorum and !self.do_view_change_quorum)
```

Comment:
I am not entirely sure about this last `!self.do_view_change_quorum` condition, but I guess our plan to broadcast DVCs woudl change the calculus here anyway

Reply:
Ah yeah my brain has already switched over to the new DVCs. It isn't necessary with the old DVC, but isn't incorrect either.